### DISCUSSION TITLE
Character shape

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#84] [Q&A] [answered]
The example is using `RotatedTranslatedShapeSettings` for the character shape. However, given the rotation is an identity quaternion, could it be using `OffsetCenterOfMassShapeSettings` instead? I tried to use it with the same position offset, but the visual result becomes different (something with the offset). Are there any other differences between the two, apart from one using a rotation and the other not?

**Comment by LeXXik** [comment]
I suppose `RotatedTranslatedShape` has a shape translated at an offset, while `OffsetCenterOfMassShape` keeps the shape in the same position, but moves its center of mass. Since we want the shape bottom to be at [0,0,0], the `OffsetCenterOfMassShape` is not suitable for character.

**Comment by jrouwe** [comment, marked as answer]
Yes, offset center of mass shape doesn't move the shape, only the center of mass, so it has no effect on CharacterVirtual
